assessments/0006-gateway-api.md

```diff
@@ -0,0 +1,240 @@
+
+
+## Website
+
+| Criteria                                    | 1   | 2   | 3   | 4   | 5   |
+| ---                                         | --- | --- | --- | --- | --- |
+| Single-source for all files                 |     |     |  ✅  |     |     |
+| Meets min website req. (for maturity level) |     |     |     |     |  ✅  |
+| Branding and design                         |     |     |     |  ✅  |     |
+| Case studies/social proof                   |     |     |  ✅  |     |     |
+| Maintenance planning                        |     |     |  ✅  |     |     |
+| A11y plan & implementation                  |     |     |     |     |  ✅  |
+| Mobile-first plan & impl.                   |     |     |     |     |  ✅  |
+| HTTPS access & HTTP redirect                |     |     |     |     |  ✅  |
+
+Scale:
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+
+### Comments
+
+**Single-source for all files**:
+
+While all files are single-sourced (that is, files are not duplicated), [the build process which governs](https://github.com/kubernetes-sigs/gateway-api/blob/master/hack/make-docs.sh) pulling in and building out some webpages is very fragile.
+
+**Meets min website req. (for maturity level)**:
+
+Project meets website requirements for its maturity level.
+
+**Branding and design**:
+
+Branding is consistently applied throughout the site. Also, the website’s typography is clean and well-suited for reading. Thus, the site looks professional! No major improvements are needed.
+
+**Case studies/social proof**:
+
+[Implementations](https://gateway-api.sigs.k8s.io/implementations/) are well-written. The project should be more active in producing blog posts.
+
+**Maintenance planning**:
+
+Having a docs+code monorepo is risky in the long term, as it couples all docs built with code builds and vice versa. If docs CI fails because Netlify is temporarily down, for example, this means that all your code builds can potentially fail as well. Coupling docs in a repository with code can also make a code repo's size expand exponentially, especially as projects pick up steam, write more blog posts (with images), and add other multimedia.
+
+**A11y plan & implementation**:
+
+The website meets the basic a11y requirements. The doc pages are readable. It is quite suitable as most website features are usable using a keyboard only. \
+
+**Mobile-first plan & impl.**:
+
+This project makes sense as it is a [mobile-first](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Responsive/Mobile_first) design. All website features are accessible from mobile -- such as the top-nav, site search, etc.
+
+**HTTPS access & HTTP redirect**:
+
+The [website](https://gateway-api.sigs.k8s.io/) is accessible via HTTPS.
+
+
+### Recommendations
+
+**Branding and design**:
+
+* We can add a _Copyright_ notice present at bottom of the page. \
```

Comment:
I see `\` at the ends of several lines. Probably best to remove them throughout as they render as backslashes when viewed on GitHub.